resources/views/backend/users/show.blade.php

```diff
@@ -1,45 +1,48 @@
 @extends('backend.layouts.main')
-@section('title', __('show.profile_user'))
+@section('title', __('user.profile_user'))
 @section('content')
 <!-- Content Wrapper. Contains page content -->
 <div class="content-wrapper">
   <!-- Content Header (Page header) -->
   <section class="content-header">
-    <h1>{{ __('show.profile_user') }}</h1>
+    <h1>{{ __('user.profile_user') }}</h1>
     <ol class="breadcrumb">
-      <li><a href="#"><i class="fa fa-show"></i> {{ __('show.admin') }}</a></li>
-      <li><a href="#">{{ __('show.users') }}</a></li>
-      <li class="active">{{ __('show.profile_user') }}</li>
+      <li><a href="{{ url('/admin') }}"><i class="fa fa-dashboard"></i> {{ __('user.admin') }}</a></li>
+      <li><a href="{{ url('/admin/users') }}">{{ __('user.users') }}</a></li>
+      <li class="active">{{ __('user.profile_user') }}</li>
     </ol>
   </section>
+  
   <!-- Main content -->
   <section class="content">
     <div class="row">
       <div class="col-md-3">
         <!-- Profile Image -->
         <div class="box box-primary">
           <div class="box-body box-profile">
-            <img class="profile-user-img img-responsive img-circle" src="" alt="User profile picture">
-            <h3 class="profile-username text-center"></h3> <center><a href=""><small>({{ __('show.edit_profile') }})</small></a></center>
-            <p class="text-muted text-center"></p>
+            <img class="profile-user-img img-responsive img-circle" src="{{ $user->avatar_url }}" alt="User profile picture">
+            <h3 class="profile-username text-center">{{ $user->name }}</h3><center></center>
+            <p class="text-muted text-center">
+              @if ($user->role == 1) {{ __('user.admin') }}
```

Comment:
tạo atribute tên roleName trong Model User
docs: 
https://laravel.com/docs/5.5/eloquent-serialization#appending-values-to-json